docs/styleguide.md

```diff
@@ -0,0 +1,24 @@
+
+List of programming style rules
+===============================
+
+(will be re-structured as it grows)
+
+- qualified imports are given the name of the module (without path),
+  not an abbreviation (`import qualified Data.Map as Map`).
+
```

Comment:
fair enough (both comments).  what i want to avoid is single-character imports like `B`, `C`, `A`.  i'll think of a rule that makes more sense.